app/pb_kits/playbook/pb_progress_simple/docs/_progress_simple_variants.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= pb_rails("progress_simple", props: { percent: 64.9, variant: "default" }) %>
+
+<br>
+
+<%= pb_rails("progress_simple", props: { percent: 64.9, variant: "positive" }) %>
+
+<br>
+
+<%= pb_rails("progress_simple", props: { percent: 64.9, variant: "negative" }) %>
```

Comment:
Could you make these different to show usage? Perhaps blue stays, positive is at 95%, and negative is 25%